distribution/ddf-common/src/main/resources/etc/templates/example.ManagedService.config

```diff
@@ -1,22 +1,42 @@
-property.string="string"
-property.boolean.true=B"true"
-property.boolean.false=B"false"
-property.int=I"10"
-property.long=L"100"
-// Floats and doubles are supported using the IEEE 754 format. See the Documentation for more details.
-property.float=F"0x41280000"
-property.double=D"0x42c83f2e"
+property.string = "string"
+property.boolean.true = B"true"
+property.boolean.false = B"false"
+property.int = I"10"
+property.long = L"100"
+# Floats and doubles are supported using the IEEE 754 format. See the Documentation for more details.
+# F"10.5"
+property.float = F"1065353216"
```

Comment:
That number corresponds to `0x3F800000` which is actually just `1.0` as a single precision float. The other ones look good though.